client/state/countries/selectors.js

```diff
@@ -0,0 +1,7 @@
+export function getCountries( state, type ) {
+	return state.countries.items[ type ];
```

Comment:
We now have a global selector folder [state/selectors](https://github.com/Automattic/wp-calypso/tree/master/client/state/selectors) for better discovery/lookup. Selectors should also be namespaced in a global manner, one selector per file.